packages/template-retail-react-app/app/pages/checkout/confirmation.jsx

```
@@ -102,6 +108,28 @@ const CheckoutConfirmation = () => {
     const CardIcon = getCreditCardIcon(order.paymentInstruments[0].paymentCard?.cardType)
 
     const submitForm = async (data) => {
+        const saveShippingAddress = async (customerId) => {
+            try {
+                const shippingAddress = order.shipments[0].shippingAddress
+                let {id, ...shippingAddressWithoutId} = shippingAddress
+                const bodyShippingAddress = {
+                    addressId: nanoid(),
+                    ...shippingAddressWithoutId
+                }
+                await createCustomerAddress.mutateAsync({
+                    body: bodyShippingAddress,
+                    parameters: {customerId: customerId}
+                })
+            } catch (error) {
+                toast({
+                    title: formatMessage(API_ERROR_MESSAGE),
+                    status: 'error',
+                    position: 'top-right',
+                    isClosable: true
+                })
```

I think having a generic error message here might not be the best UX because we are making 2 and if the user see's "Something went wrong" they wouldn't know if it was the create user call that failed or the create address call. 

We have 2 options here:

1. Get specific about the error, e.g. say something like said to set default address.
2. Fail silently aka don't use a toast don't catch any error. 

I think the later might be the best thing to do in this situation because if we tell them that the address wasn't saved, we aren't immediately letting them know how to resolve it or try again. In the second option, what they don't know won't hurt them. They might just have to add an address on their second order.

---

@bendvc I agree. Will fail silently/don't do anything in the catch (error) clause :)